examples/suit_update/README.md

```diff
@@ -483,10 +484,249 @@ success, the application reboots on the new slot:
     Running from slot 1
     ...
 
-
 The slot number should have changed from after the application reboots.
 You can do the publish-notify sequence several times to verify this.
 
+## Testing Without Hardware
+[testing-without-hardware]: #testing-without-hardware
+
+The SUIT update example is compatible with the native application to try out the
+update process. While it is not possible to update the running code in the
+application, the workflow can still be used to update an in-memory storage.
+
+### Description
+
+When building the example application for the native target, the firmware update
+capability is removed. Instead two in-memory slots are created that can be
+updated with new payloads. These act as a demonstrator for the SUIT
+capabilities.
+
+### Workflow
+
+While the above examples use make targets to create and submit the manifest,
+this workflow aims to provide a better view of the SUIT manifest and signature
+workflow. Because of this the steps below use the low level scripts to manually
+creates a payload and manifest and sign it.
+
+#### Setting up networking
+
+To deliver the payload to the native instance, a network connection between a
+coap server and the instance is required.
+
+First a bridge with two tap devices is created:
+
+```console
+$ sudo dist/tools/tapsetup/tapsetup -c
+```
+
+This creates a bridge called `tapbr0` and a `tap0` and `tap1`. These last two
+tap devices are used by native instances to inject and receive network packets
+to and from.
+
+On the bridge device `tapbr0` an routable IP address is added such as
```

Comment:
Is this explaining or asking to do something and it should be "add a routable IP address..."